doc/classes/Performance.xml

```diff
@@ -217,7 +217,7 @@
 			Number of islands in the 3D physics engine.
 		</constant>
 		<constant name="AUDIO_OUTPUT_LATENCY" value="26" enum="Monitor">
-			Output latency of the [AudioServer].
+			Output latency of the [AudioServer]. Equivalent to calling [gdscript]AudioServer.get_output_latency()[/gdscript], it is not recommended to call this every frame.
```

Comment:
There's no `[gdscript]` inline code tag, it would be just `[code]`. But this should use a direct hyperlink to the method.
```suggestion
			Output latency of the [AudioServer]. Equivalent to calling [method AudioServer.get_output_latency], it is not recommended to call this every frame.
```

Reply:
Thanks!